qa/tasks/deepsea.py

```diff
@@ -363,8 +404,64 @@ def _introspect_roles(self, deepsea_ctx):
         for var in report_vars:
             self.log.info("{} == {}".format(var, deepsea_ctx[var]))
 
+    def _master_python_version(self, py_version):
+        """
+        Determine if a given python version is installed on the Salt Master
+        node.
+        """
+        python_binary = 'python{}'.format(py_version)
+        installed = True
+        try:
+            self.master_remote.run(args=[
+                'type',
+                python_binary,
+                run.Raw('>'),
+                '/dev/null',
+                run.Raw('2>&1'),
+                ])
+        except CommandFailedError:
+            installed = False
+        if installed:
+            self.master_remote.run(args=[
+                python_binary,
+                '--version'
+                ])
+        else:
+            self.log.info(
+                '{} not installed on master node'.format(python_binary)
+                )
+        return installed
+
+    def _maybe_apply_alternative_defaults(self):
```

Comment:
Is having *_maybe* prefix for method some kind of convention? I think for this method it is very weird, please, rename to _apply_alternative_defaults.

Reply:
Yes, it's a convention. The "maybe" signals that there might not be any alternative defaults, but we call the function in either case.

Reply:
> Yes, it's a convention. The "maybe" signals that there might not be any alternative defaults, but we call the function in either case.

I would vote against this kind of conventions, since any method cad do or not some action.